todo_app/data/session_items.py

```diff
@@ -1,69 +1,42 @@
-from flask import session
-
-_DEFAULT_ITEMS = [
-    { 'id': 1, 'status': 'Not Started', 'title': 'List saved todo items' },
-    { 'id': 2, 'status': 'Not Started', 'title': 'Allow new items to be added' }
-]
-
-
-def get_items():
-    """
-    Fetches all saved items from the session.
-
-    Returns:
-        list: The list of saved items.
-    """
-    return session.get('items', _DEFAULT_ITEMS.copy())
-
-
-def get_item(id):
-    """
-    Fetches the saved item with the specified ID.
-
-    Args:
-        id: The ID of the item.
-
-    Returns:
-        item: The saved item, or None if no items match the specified ID.
-    """
-    items = get_items()
-    return next((item for item in items if item['id'] == int(id)), None)
-
-
-def add_item(title):
-    """
-    Adds a new item with the specified title to the session.
-
-    Args:
-        title: The title of the item.
-
-    Returns:
-        item: The saved item.
-    """
-    items = get_items()
-
-    # Determine the ID for the item based on that of the previously added item
-    id = items[-1]['id'] + 1 if items else 0
-
-    item = { 'id': id, 'title': title, 'status': 'Not Started' }
-
-    # Add the item to the list
-    items.append(item)
-    session['items'] = items
-
-    return item
-
-
-def save_item(item):
-    """
-    Updates an existing item in the session. If no existing item matches the ID of the specified item, nothing is saved.
-
-    Args:
-        item: The item to save.
-    """
-    existing_items = get_items()
-    updated_items = [item if item['id'] == existing_item['id'] else existing_item for existing_item in existing_items]
-
-    session['items'] = updated_items
-
-    return item
+from flask import session,render_template
+from flask.wrappers import Response
+from todo_app.flask_config import Config
+import requests
+import json
+
+def getTodoCard():
+    todourl="https://api.trello.com/1/lists/610beb5a99354405b1f4ec6f/cards/?key="+Config.TRELLO_KEY+"&token="+Config.TRELLO_TOKEN
+    response = requests.get(todourl)
+    data = response.text
+    cardname=json.loads(data)
+    return cardname
```

Comment:
In terms of clean code, this returns a list of cards right? So it might be better to name it `getTodoCards`, or `get_todo_cards`. snake_case is common for Python method names.

Similarly `cardname` is not a helpful variable name here.

One other thing, the response object has a `.json()` method which is similar to what you do on lines 10 and 11.

Reply:
updated name